src/test/java/org/jfrog/hudson/pipeline/integration/CommonITestsPipeline.java

```diff
@@ -210,8 +211,10 @@ void downloadNonExistingBuildTest(String buildName) throws Exception {
      * Verify that we don't download files with same sha and different build name and build number.
      */
     void downloadByShaAndBuildTest(String buildName) throws Exception {
-        Set<String> expectedDependencies = Sets.newHashSet("a3");
-        Set<String> unexpected = Sets.newHashSet("a4", "a5");
+        Set<String> expectedDependencies = Collections.singleton("a3");
+        Set<String> unexpected = new HashSet<>();
+        unexpected.add("a4");
+        unexpected.add("a5");
```

Comment:
For good order, let's do this in all places whereby there is more than one element in a set in this file.
```suggestion
        Collections.addAll(unexpected, "a4", "a5");
```